modules/Bio/EnsEMBL/Variation/Utils/VEP.pm

```diff
@@ -478,12 +479,7 @@ sub parse_ensembl {
         my $so_term;
 
         # convert to SO term
-        my %terms = (
-            INS  => 'insertion',
-            DEL  => 'deletion',
-            TDUP => 'tandem_duplication',
-            DUP  => 'duplication'
-        );
+        my %terms = %SV_SO_TERMS;
```

Comment:
A SV based on `BND` here won't include the breakends. Do we want to support the `BND` here?

Reply:
I tested with a variant like this - `1 7936271 7936271 BND + bnd` and from `parse_ensembl` got -
```
{
                 'class_SO_term' => 'chromosome_breakpoint',
                 'start' => '7936271',
                 'adaptor' => undef,
                 'variation_name' => 'bnd',
                 'chr' => '1',
                 'strand' => 1,
                 'end' => '7936271'
               }, 'Bio::EnsEMBL::Variation::StructuralVariationFeature' );
```

seems to work.